security.rst

```diff
@@ -300,8 +302,20 @@ you are anonymous:
 .. image:: /_images/security/anonymous_wdt.png
    :align: center
 
+It will also allow a request to access a form login without being authenticated as a
+unique user (otherwise an infinite redirection loop would happen asking the user to
+authenticate while trying to doing so).
```

Comment:
-1 on this, I believe it's discussed when implementing a form login.

Reply:
The redirection loop is always a WTF moment for ppl thinking they can remove this anonymous preconfiguration in favor of something else. The sentence is short enough without being complex, again let's try to be clear about it as soon as we can instead of hoping they can find the right page and section later (for the ref it's line 600+ in the form login rst doc). Maybe you can help with the wording though? Maybe `It is convenient to allow` instead of `It will also`? Or we can try to fix the form login article instead?